pymc/distributions/continuous.py

```diff
@@ -1019,8 +1019,41 @@ def logcdf(value, mu, lam, alpha):
             msg="mu > 0, lam > 0, alpha >= 0",
         )
 
-    def icdf(mu, lam):
-        return stats.invgauss(mu=mu, scale=lam)
+    def icdf(value, mu, lam, alpha=0):
```

Comment:
We don't provide defaults in these functions, it's always done through the `dist` classmethod

```suggestion
    def icdf(value, mu, lam, alpha):
```

Reply:
adjusted